pkg/cmd/server/kubernetes/node_config.go

```diff
@@ -280,3 +288,30 @@ func BuildKubernetesNodeConfig(options configapi.NodeConfig) (*NodeConfig, error
 
 	return config, nil
 }
+
+// ensureDockerAPIVersionForParallelImagePulls returns true if the docker API version
+// is >=1.21, else false.
+func ensureDockerAPIVersionForParallelImagePulls(dh *docker.Helper, version string) bool {
+	dockerClient, _, err := dh.GetClient()
+	if err != nil {
+		return false
+	}
+
+	env, err := dockerClient.Version()
+	if err != nil {
+		return false
+	}
+
+	serverVersionString := env.Get("ApiVersion")
+	serverVersion, err := dockerclient.NewAPIVersion(serverVersionString)
+	if err != nil {
+		return false
+	}
+
+	minimumVersion, err := dockerclient.NewAPIVersion(version)
+	if err != nil {
+		return false
```

Comment:
glog.Warningf("Unable to determine minimal Docker server version from %q.\n%v", serverVersionString, err)